### PR TITLE
SW-275 - Adding custom header for matched route name to nginx log

### PR DESCRIPTION
**Pull request by dragoonis**
… inspec tests

### CODE REVIEW

File: opg-nginx-1604/tests/controls/default.rb [hunk 1]
```diff
@@ -47,3 +47,18 @@
...
 end
+
+control 'Sirius ZF Matched Route Name Exists' do
+  impact 1
+  title 'Sirius ZF Matched Route Name Exists'
+  desc 'Sirius ZF Matched Route Name Exists'
+  # app severname will not log for static requests so no logs will be produced.
+  # Workaround by sending to default host which drops connection but does
+  # add 444 request to logs.
+  describe command('curl -k "https://localhost/matched_route_path?myquerystring" -H "X-Sirius-ZF-Matched-Route-Name: SomeRouteValue"') do
```
Comment (carinadigital) [inline opg-nginx-1604/tests/controls/default.rb]: Why not combine your test with the one above?

File: opg-nginx-1604/docker/etc/nginx/nginx.conf [hunk 2]
```diff
@@ -64,6 +64,7 @@ http {
                              '"query_string": "$query_string", '
                              '"uri": "$uri", '
                              '"request_path": "$request_path" '
+                             '"zf_matched_route_name": "$http_x_sirius_zf_matched_route_name", '
```
Comment (carinadigital) [inline opg-nginx-1604/docker/etc/nginx/nginx.conf]: Have you tested what the logs look like when you don't send the header through?
Anything appearing in nginx error log?

File: opg-nginx-1604/tests/controls/default.rb (hunk 1 above)
Comment (dragoonis) [reply opg-nginx-1604/tests/controls/default.rb]: done

File: opg-nginx-1604/docker/etc/nginx/nginx.conf (hunk 2 above)
Comment (dragoonis) [reply opg-nginx-1604/docker/etc/nginx/nginx.conf]: confirmed. it defaults to blank string.

**Pull request merged**